spec/requests/sidebar_sections_controller_spec.rb

```diff
@@ -135,6 +135,9 @@
     fab!(:section_link_2) do
       Fabricate(:sidebar_section_link, sidebar_section: sidebar_section, linkable: sidebar_url_2)
     end
+    let(:community_section) do
+      SidebarSection.where(section_type: SidebarSection.section_types[:community]).first
+    end
```

Comment:
I'm wondering why we have to add this here when it isn't being used in the test block.

Reply:
Ah, my mistake when splitting code, will remove it

Reply:
No worries, was guessing the it probably isn't necessary.